src/Controls/tests/UITests/Tests/Issues/Issue19956.cs

```diff
@@ -74,14 +83,40 @@ public void BottomInsetsSetCorrectly()
     }
 
     static void ScrollToBottom(AppiumApp app)
-    {
-        var actions = new TouchAction(app.Driver);
-        // scroll up once to trigger resetting content insets
-        actions.LongPress(null, 5, 300).MoveTo(null, 5, 450).Release().Perform();
-        actions.LongPress(null, 5, 400).MoveTo(null, 5, 100).Release().Perform();
-
-        actions.LongPress(null, 5, 400).MoveTo(null, 5, 100).Release().Perform();
-        actions.LongPress(null, 5, 400).MoveTo(null, 5, 100).Release().Perform();
+	{
+		OpenQA.Selenium.Appium.Interactions.PointerInputDevice touchDevice = new OpenQA.Selenium.Appium.Interactions.PointerInputDevice(PointerKind.Touch);
+		
+		var scrollSequence1 = new ActionSequence(touchDevice, 0);
```

Comment:
Can you consolidate `LongPress` behind something re-usable? I think we can add an extension method to `Iapp` that we can reuse across all these tests yea?